data/json/items/gun/77mm_jap.json

```diff
@@ -0,0 +1,80 @@
+[
+  {
+    "id": "type99",
+    "copy-from": "rifle_manual",
+    "looks_like": "modular_ar15",
+    "type": "GUN",
+    "name": { "str": "7.7mm battle rifle" },
+    "description": "A stout and rugged battle rifle that traces its origins to the hands of imperial Japanese forces and saw extensive use in the Pacific theatre of the Second World War, this bolt-action firearm shoots the 7.7x58mm Arisaka cartridge.  Powerful, dependable, and elegantly built, Japan’s 20th-century infantry rifle may yet find service guarding the lives of her former enemies.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "type99",
+        "name": { "str_sp": "Arisaka Type 99" },
+        "description": "The standard service arm of the imperial Japanese army during the Second World War, chambered for the powerful 7.7x58mm cartridge and built to replace the weaker Type 38, the Type 99 is a sturdy and formidable bolt-action rifle with a Mouser-inspired action renowned for its strength.  Though the rifle was historically sneered at in the US due to extensive anti-Japanese propaganda and the lower quality of later-production guns, the weapon is, by most metrics, a finely produced and effective shoulder arm.  This particular Arisaka is an early-war example with anti-aircraft sights and an intact 6-pettle chrysanthemum seal.",
+        "weight": 1
+      },
+      {
+        "id": "type99ld",
```

Comment:
Does the ld here mean "last-ditch," like in the description 🤔

Reply:
Sorry for taking a while to get to these, I was away for a stretch. “ld” does stand for last-ditch. Felt like the id would have been too huge if I didn’t abbreviate it.